packages/k8s/k0s/build.yaml

```diff
@@ -0,0 +1,40 @@
+requires:
+- name: "toolchain-go-ubuntu"
+  category: "development"
+  version: ">=0"
+env:
+  - K0S_VERSION=v{{ regexReplaceAll "\\+\\d+$" .Values.version "" }}+k0s.{{.Values.k0s_version}}
+  {{$arch:=(default "amd64" .Values.arch)}}
+  {{ if eq $arch "arm" }}
+  - ARCH=arm64
+  {{ else }}
```

Comment:
do we need all this arch stuff? the installer seems to know which arch its running on so it should not be necessary to do it ourselves

Reply:
good observation